worlds/osu/Client.py

```diff
@@ -558,6 +559,41 @@ async def auto_get_last_scores(ctx, mode=''):
             ctx.last_scores.pop(0)
         check_location(ctx, score)
 
+# This is the non-silent version of the function above where this one is used in the update command
+async def get_last_scores(self, mode=''):
+    # Make URl for the request
+    try:
+        request = f"https://osu.ppy.sh/api/v2/users/{os.environ['PLAYER_ID']}/scores/recent?include_fails=1&limit=100"
+    except KeyError:
+        self.output('No Player ID')
+        return
+    # Add Mode to request, otherwise it will use the user's default
+    if mode:
```

Comment:
Doesn't check if the gamemode is valid. Current Update Command uses:
`if mode and mode.lower() in self.mode_names.keys():`
`  request += f"&mode={self.mode_names[mode.lower()]}"`